MAINTAINERS.yml

```diff
@@ -372,13 +364,8 @@ Documentation:
         - jocelyn-li
         - mbolivar-nordic
     files:
-        - doc/*
-        - doc/static/
-        - doc/templates/
-        - doc/custom-doxygen/
-        - doc/scripts/
+        - doc/
```

Comment:
there should be a distinction between doc infra and subsystem and content documentation.

Reply:
I'm assuming that the "area Documentation" is the "doc infra" you referred above, and the "doc infra" should take `doc/CMakeLists.txt` and other build related files.

There are multiple subsystems taking its own docs but I don't see any group which I can assign `doc/*.rst`.  Where / who should take `doc/*.rst`?